src/preferences/dialog/dlgprefrecord.cpp

```diff
@@ -201,7 +212,12 @@ void DlgPrefRecord::slotResetToDefaults() {
 
     // 4GB splitting is the default
     comboBoxSplitting->setCurrentIndex(4);
+
+    // Sets 'Create a CUE file' checkbox value
     CheckBoxRecordCueFile->setChecked(kDefaultCueEnabled);
+
+    // Sets 'Enable File Annotation in CUE file' checkbox value
+    CheckBoxUseCueFileAnnotation->setChecked(false);
```

Comment:
Can you capture that default value in a `kConstant` like for line 217?

Reply:
Sure, I did this originally and then scrapped it when I realized I could keep it as `false`. This builds fine on my end. I was curious why `kDefaultCueEnabled` was necessary in this case but I could not find it anywhere else in the code. So if you can explain why we capture the default in this way (as opposed to say `getValueString`), I would greatly appreciate it!

Reply:
This helps with readability mainly -  all defaults get listed at the top of the file, and this helps to understand why this `false` is being set to `CheckBoxUseCueFileAnnotation`.

Reply:
That makes sense, thanks!